docs/monster-util/isAuthorizedTopicForCrossSiteMessaging.md

```diff
@@ -0,0 +1,19 @@
+"title: isAuthorizedTopicForCrossSiteMessaging()
```

Comment:
Rename the file to `isAuthorizedTopicForCrossSiteMessaging().md` to follow the naming convention, and add it to the file `docs/mkdocs/mkdocs.yml` under the `monster.util` entry, I think that may fix the Circle error.

Reply:
Ah good eye/catch 👓